src/GraphQL/Introspection/__Field.cs

```diff
@@ -28,7 +28,7 @@ public __Field()
                 {
                     // We have to iterate over all schema types because FieldType has no reference to the GraphType to which it belongs.
                     var fieldOwner = context.Schema.AllTypes.OfType<IComplexGraphType>().Single(t => t.Fields.Contains(context.Source));
-                    if (fieldOwner is IImplementInterfaces implementation && implementation.ResolvedInterfaces != null)
+                    if (fieldOwner is IImplementInterfaces {ResolvedInterfaces: { }} implementation)
```

Comment:
It's harder to understand. Please revert.

Reply:
Agreed